README.md

```diff
@@ -1,31 +1,39 @@
 CellMincer
 ===========
 
-CellMincer is a software package for learning self-supervised denoising models for voltage-imaging movies.
+CellMincer is a self-supervised machine learning framework for voltage imaging denoising models. A visual comparison of voltage imaging data before and after CellMincer below:
 
-Installation
-============
+![Alt Text](./docs/source/_static/graphics/raw_denoised_traces.gif)
 
-```
-git clone https://github.com/broadinstitute/CellMincer.git
-pip install -e CellMincer/
-```
+A schmatic of Optopatch data and CellMincer's high-level architecture below:
+
+![Alt Text](./docs/source/_static/graphics/fig1-cellmincer-schem.png)
 
-Modules
-=======
+# Documentation and resources
 
-CellMincer provides the following tools:
+Coming soon!
```

Comment:
You can host the documentation on GitHub pages using GitHub actions, see below:
https://coderefinery.github.io/documentation/gh_workflow/

Here's an example:
https://cellarium-ai.github.io/cellarium-ml/

For now, make an issue.